Data/ByteString/Lazy/Internal.hs

```diff
@@ -142,6 +142,7 @@ instance Data ByteString where
 -- Packing and unpacking from lists
 
 packBytes :: [Word8] -> ByteString
+packBytes [] = mempty
```

Comment:
Sorry for a minor nitpicking, but I would rather use `Empty` instead of `mempty`: it avoids an indirection and is shorter at the same time.

Reply:
Sure, I just wrote `mempty` pretty automatically.